snow/validators/traced_state.go

```diff
@@ -51,7 +53,7 @@ func (s *tracedState) GetCurrentHeight(ctx context.Context) (uint64, error) {
 }
 
 func (s *tracedState) GetSubnetID(ctx context.Context, chainID ids.ID) (ids.ID, error) {
-	ctx, span := s.tracer.Start(ctx, s.getValidatorSetTag, oteltrace.WithAttributes(
+	ctx, span := s.tracer.Start(ctx, s.getSubnetIDTag, oteltrace.WithAttributes(
```

Comment:
super nit - but I think we could make this its own PR

Reply:
https://github.com/ava-labs/avalanchego/pull/3542/files